phantomjs/bridge.js

```diff
@@ -9,7 +9,7 @@
 /*global QUnit:true, alert:true*/
 (function (factory) {
   if (typeof define === 'function' && define.amd) {
-    require(['qunit'], factory);
+    define('phantom-bridge', ['qunit'], factory);
```

Comment:
Once the module is named it must be required by that exact name. [The recommendation from RequireJS](http://requirejs.org/docs/api.html#modulename) is to not code in names:

>You can explicitly name modules yourself, but it makes the modules less portable -- if you move the file to another directory you will need to change the name. It is normally best to avoid coding in a name for the module and just let the optimization tool burn in the module names. The optimization tool needs to add the names so that more than one module can be bundled in a file, to allow for faster loading in the browser.

For an example of using `grunt-contrib-qunit`, `QUnit` and `RequireJS`, you can either go with the solution mentioned in this [Stackoverflow answer](http://stackoverflow.com/a/29890894/3904791) or with the amd test in `QUnit` (this [two](https://github.com/jquery/qunit/blob/master/test/amd.html) [files](https://github.com/jquery/qunit/blob/master/test/amd.js))